hips/draw/simple.py

```diff
@@ -30,13 +32,21 @@ def draw_sky_image(geometry: WCSGeometry, tiles: Generator[HipsTile, Any, Any],
         A list of HipsTile
     hips_survey : `~hips.HipsSurveyProperties`
         HiPS survey properties
+    tile_format : `str`
+        Format of HiPS tile
 
     Returns
     -------
     np.ndarray
         Returns a numpy array containing all HiPS tiles projected onto it
     """
-    image = np.zeros(geometry.shape)
+    if tile_format == 'jpg':
+        image = np.zeros((geometry.shape.width, geometry.shape.height, 3))
```

Comment:
Nitpick comment: I'd suggest to just compute a local `shape` variable here in the if / else block, and then to put `image = np.zeros(shape)` below, i.e. not duplicate the `np.zeros` code multiple times.